utils/nctl/sh/scenarios/upgrade_after_emergency_upgrade_test_pre_1.5.sh

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/utils/main.sh
+source "$NCTL"/sh/views/utils.sh
+source "$NCTL"/sh/assets/upgrade.sh
+source "$NCTL"/sh/scenarios/common/itst.sh
+source "$NCTL"/sh/node/svc_"$NCTL_DAEMON_TYPE".sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that performs an emergency restart on the network,
+# and then also performs a regular upgrade after.
+# It also simulates social consensus on replacing the original validators (nodes 1-5)
+# with a completely new set (nodes 6-10).
+#
+# Arguments:
+#   `timeout=XXX` timeout (in seconds) when syncing. Default=300 seconds.
+#   `version=X_Y_Z` new protocol version to upgrade to. Default=2_0_0.
+#   `version2=X_Y_Z` new protocol version to upgrade to. Default=3_0_0.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Upgrade after emergency upgrade test begins"
```

Comment:
```suggestion
    log "Upgrade after emergency upgrade test pre 1.5 begins"
```

to prevent ambiguity on the console output between this test and the one in `upgrade_after_emergency_upgrade_test.sh`

Reply:
Addressed in 2bd2566